lib/db/players.js

```diff
@@ -87,41 +51,34 @@ async function playerInfoAll(playerIds) {
     `SELECT * 
 		FROM data.players_info_expanded_all 
 		WHERE player_id=ANY($1)
-		ORDER BY player_id, valid_from`,
+		ORDER BY player_name, valid_from`,
     [normalizeCommaSeparatedParam(playerIds)]
   );
 }
 
 async function taggedPlayers() {
   return await all(
-    `select p.player_id, player_name, modification, pm.valid_from, t.team_id, t.nickname
-		from data.player_modifications pm 
-		join data.players p on p.player_id = pm.player_id
-		join data.team_roster tr on p.player_id = tr.player_id
-		join data.teams t on t.team_id = tr.team_id
-		where p.valid_until is null
-		and tr.valid_until is null
-		and t.valid_until is null
-		and p.deceased = false
-		and pm.valid_until is null
-		order by valid_from`
+    `SELECT * 
+		FROM data.players_info_expanded_all 
+		WHERE valid_until IS NULL AND modifications IS NOT null order by player_name`
   );
 }
 
 // The player encyclopedia page is { playerName, playerSlug, isIncinerated } and the teams page is { teamFullName, teamSlug, teamEmoji, teamMainColor } I think
 
 async function allPlayers(includeShadows = false) {
-  let view = includeShadows
-    ? 'data.players_extended_current'
-    : 'data.players_current';
-
   return await all(
-    `SELECT player_id, player_name, gameday_from, season_from, deceased, team_id, team, position_id, position_type
-		FROM ${view}
+    `SELECT *
+		FROM data.players_info_expanded_all 
+		WHERE valid_until IS NULL
+		AND position_id IS NOT NULL
+		${includeShadows ? 'AND position_type_id < 2' : ''}
```

Comment:
I think this ternary is backwards, you want to filter if shadows are not included

Reply:
I believe you're right